libs/solid/skeleton/src/lib/components/feedback/feedback.component.scss

```diff
@@ -8,6 +8,10 @@ mat-form-field {
   }
 }
 
+::ng-deep .mat-button-focus-overlay {
+  background-color: transparent;
```

Comment:
Wird hier das `!important` nicht gebraucht?

(Ist nur eine Frage - ich weiß es nicht, aber es war in der anderen Komponente so. Wenn es ohne geht, dann ist es gut)